doc/man3/EVP_PKEY_new.pod

```diff
@@ -23,6 +35,25 @@ EVP_PKEY_up_ref() increments the reference count of B<key>.
 EVP_PKEY_free() decrements the reference count of B<key> and, if the reference
 count is zero, frees it up. If B<key> is NULL, nothing is done.
 
+EVP_PKEY_new_priv_key() allocates a new B<EVP_PKEY>. If B<e> is non-NULL then
+the new B<EVP_PKEY> structure is associated with the engine B<e>. The B<type>
+argument indicates what kind of key this is. The value should be a NID for a
+public key algorithm that supports raw private keys, i.e. one of
+B<EVP_PKEY_CMAC>, B<EVP_PKEY_HMAC>, B<EVP_PKEY_POLY1305>, B<EVP_PKEY_SIPHASH>,
+B<EVP_PKEY_X25519>, B<EVP_PKEY_ED25519>, B<EVP_PKEY_X448> or B<EVP_PKEY_ED448>.
+B<key> points to the raw private key data for this B<EVP_PKEY> which should be
+of length B<keylen>. The length should be appropriate for the type of the key.
+The public key data will be automatically derived from the given private key
+data.
```

Comment:
The RETURNS sections needs to be updated.

Reply:
Fixed. Also fixed HISTORY while I was at it.